_posts/2021-09-26-where-the-store-is-stored.md

```diff
@@ -0,0 +1,292 @@
+---
+title: Where is my store stored?
+
+draft: false
+
+page : false
+
+image: /assets/img/where-is-store-stored-illustration.png 
+
+excerpt: How looking at where stores are stored in our Javascript codebases is
+useful to understand the constraints that come with those locations and how the 
+stores can be used.
+
+---
+
+<img
+  alt="Abstract illustration of a person holding two puzzle pieces"
+  src="/assets/img/where-is-store-stored-illustration.png"
+  style="max-height: 480px; margin: 2rem auto"
+/>
+
+Some of us have concepts in mind that they keep returning to over an over, as if
+they haven't completely internalized them. For me as a frontend software
+engineer, it's all about the concepts of *State* and *Store*. I keep looking at
+technical issues through this prism - current and previous colleagues know this
+all too well (sorry colleagues!). I've actually already [written about
+it](/tech/posts/2020-11-22-state-store-in-frontend-codebases) in the past.
+
+I have changed companies a few month ago to join BlaBlaCar and the very cool
```

Comment:
A four line sentence is a bit long, do you think splitting it can improve the reading ?